src/components/App/App.js

```diff
@@ -1,46 +1,57 @@
-// import movieData from '../../movieData';
 import './App.css';
 import React, { Component } from 'react';
 import Header from '../Header/Header';
 import MoviesContainer from '../MoviesContainer/MoviesContainer';
 import MovieDetailer from '../MovieDetailer/MovieDetailer';
-import { allMovies, singleMovie } from '../../apiCalls'
+import { allMovies, singleMovie, singleMoviesVideos } from '../../apiCalls'
 
 class App extends Component {
   constructor() {
     super();
     this.state = {
       movies: [],
-      movieID: 0,
-      movie: {}
+      movieID: '',
```

Comment:
Setting default as string solves display problem of 00s being rendered by MovieDetailer. Would be good, however, to try  on a future refactor to resolve the type conflict